$OpenBSD: patch-make-target-contrib_sh,v 1.4 2013/12/10 17:37:54 joshe Exp $

Allow the makefile target to be overridden so that the contribs may be
built first (perhaps under USE_SYSTRACE=Yes) and the tests run later
in do-regress.

--- make-target-contrib.sh.orig	Sat Nov 30 06:28:18 2013
+++ make-target-contrib.sh	Sun Dec  8 18:03:12 2013
@@ -73,7 +73,7 @@ for i in $contribs_to_build; do
     test -f obj/asdf-cache/$i/test-passed.test-report && rm obj/asdf-cache/$i/test-passed.test-report
     mkdir -p obj/asdf-cache/$i/
     # hack to get exit codes right.
-    if $GNUMAKE -C contrib/$i test < /dev/null 2>&1 && touch obj/asdf-cache/$i/test-passed.test-report ; then
+    if $GNUMAKE -C contrib/$i ${SBCL_CONTRIB_TARGET:-test} < /dev/null 2>&1 && touch obj/asdf-cache/$i/test-passed.test-report ; then
 	:
     else
 	exit $?
